views/index.ejs

```diff
@@ -1,96 +1,123 @@
 <!DOCTYPE html>
 <html>
 
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="description" content="Baat Cheet is an anonymous chat app">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title><%= title %></title>
-    <%- include('head_imports'); %>
-  </head>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="description" content="Baat Cheet is an anonymous chat app">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title><%= title %></title>
+  <%- include('head_imports'); %>
+</head>
 
-  <body>
-    <%- include('user', {title: title}); %>
+<body>
+  <%- include('user', {title: title}); %>
 
-    <div class="wrapper">
-      <div class="app-container">
-        <div class="left">
-          <div class="form-group mb-0 px-3 pt-4">
-            <input type="text" id="searchId" placeholder="Search..." class="searchtext form-control" />
-            <hr class="mb-0"/>
-          </div>
-          <!-- Overflow style div -->
-          <div class="room-list">
-            <div class="card-columns m-2" name="people">
-              <div class="card active person" data-chat="lobby" id="lobby" onclick="showRoom(this)">
-                <div class="card-body">
-                  <h5 class="card-title name">Lobby</h5>
-                  <p class="card-text preview">Central Lobby</p>
-                </div>
+  <div class="wrapper">
+    <div class="app-container">
+      <div class="left">
+        <div class="form-group mb-0 px-3 pt-4">
+          <input type="text" id="searchId" placeholder="Search..." class="searchtext form-control" />
+          <hr class="mb-0" />
+        </div>
+        <!-- Overflow style div -->
+        <div class="room-list">
+          <div class="card-columns m-2" name="people">
+            <div class="card active person" data-chat="lobby" id="lobby" onclick="showRoom(this)">
+              <div class="card-body">
+                <h5 class="card-title name">Lobby</h5>
+                <p class="card-text preview">Central Lobby</p>
```

Comment:
This all is also unnecessary changes, please use `save without formatting in your editor or change formatting rules of your editor.